core/templates/pages/collection-editor-page/editor-tab/collection-node-creator.directive.ts

```diff
@@ -148,9 +152,9 @@ angular.module('oppia').directive('collectionNodeCreator', [
             }
 
             // Create a new exploration with the given title.
-            $http.post('/contributehandler/create_new', {
-              title: title
-            }).then(function(response) {
+            CollectionNodeCreatorBackendApiService.createNewExploration(
```

Comment:
This is not creating any node, I believe we already have a service to create exploration, can you please check and use that?